### ISSUE TITLE
Issue while using django-haystack and django-machina with haystack.backends.simple_backend.SimpleEngine

### ISSUE THREAD

**Issue by clincher** [ellmetha/django-machina#131]
I'm using django-machina but I don't need to use search engine yet, but when I'm trying to search ANYTHING while using regular haystack.backends.simple_backend.SimpleEngine, I receive such SQL queries in logs:

`SELECT forum_conversation_post.id,  forum_conversation_post.created,  forum_conversation_post.updated,  forum_conversation_post.topic_id,  forum_conversation_post.poster_id,
forum_conversation_post.poster_ip,  forum_conversation_post.anonymous_key,  forum_conversation_post.subject,  forum_conversation_post.content,  forum_conversation_post.username,  forum_conversation_post.approved,  forum_conversation_post.enable_signature,  forum_conversation_post.update_reason,  forum_conversation_post.updated_by_id, forum_conversation_post.updates_count,  forum_conversation_post._content_rendered FROM forum_conversation_post WHERE (forum_conversation_post.anonymous_key LIKE '%[1]>%' ESCAPE '\' OR forum_conversation_post.subject LIKE '%[1]>%' ESCAPE '\' OR forum_conversation_post.content LIKE '%[1]>%' ESCAPE '\' OR forum_conversation_post.username LIKE '%[1]>%' ESCAPE '\' OR forum_conversation_post.update_reason LIKE '%[1]>%' ESCAPE '\' OR forum_conversation_post._content_rendered LIKE '%[1]>%' ESCAPE '\') ORDER BY forum_conversation_post.created ASC`

In short, it just changes any search phrase to "[1]>" string on the SQL level.

I've spent many hours tying to figure out what's wrong with it, but can't understand why does it happen and how to solve the issue. I didn't figure out if it's a machina issue or haystack tho
